surveying_dust_structure_via_GALEX_MIS_pt1/surveying_dust_structure_via_GALEX_MIS_pt1.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<p>This is a beautiful image! What a great pick for this notebook :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>